lms/templates/onedrive.html.jinja2

```diff
@@ -0,0 +1,15 @@
+{% extends "lms:templates/base.html.jinja2" %}
+
+{% block content %}
+  <div id="app"></div>
+{% endblock %}
+
+{% block styles %}
+  {% for url in  asset_urls("frontend_apps_css") %}
+    <link rel="stylesheet" href="{{ url }}">
+  {% endfor %}
+{% endblock %}
```

Comment:
Based on a Slack discussion, my understanding is that the only content which is actually needed in this file is the OneDrive `<script>` at the bottom. The `styles` and `content` block can be omitted or made empty.